src/main/kotlin/org/rust/ide/annotator/fixes/EncloseConstExprInBracesFix.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.annotator.fixes
+
+import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import com.intellij.psi.util.parentOfType
+import org.rust.lang.core.psi.RsExpr
+import org.rust.lang.core.psi.RsPsiFactory
+
+class EncloseConstExprInBracesFix(element: PsiElement) : LocalQuickFixAndIntentionActionOnPsiElement(element) {
```

Comment:
```suggestion
class EncloseExprInBracesFix(element: PsiElement) : LocalQuickFixAndIntentionActionOnPsiElement(element) {
```

Just to make a fix a bit more general since it's doesn't require anything special from expression expression element